documentation.md

```diff
@@ -109,7 +109,22 @@ const bodypose = ml5.bodypose(?options, ?callback);
   }
   ```
 
-  More info on options [here](https://github.com/tensorflow/tfjs-models/tree/master/pose-detection/src/movenet#create-a-detector).
+  More info on options for MoveNet [here](https://github.com/tensorflow/tfjs-models/tree/master/pose-detection/src/movenet#create-a-detector).
```

Comment:
```suggestion
  [More info on options for MoveNet](https://github.com/tensorflow/tfjs-models/tree/master/pose-detection/src/movenet#create-a-detector).
```

For accessibility / screen reader support, it's best pratice to include descriptive text for the link, rather than "here" or "click here".